src/App.js

```diff
@@ -1,16 +1,16 @@
-import React from 'react';
-import './App.css';
+import React from "react";
+import "./App.css";
 
-import BusinessList from '../BusinessList/BusinessList';
-import SearchBar from '../SearchBar/SearchBar';
+import BusinessList from "./components/BusinessList/BusinessList";
+import SearchBar from "./components/SearchBar/SearchBar";
```

Comment:
you weren't importing these right, see how they're inside the components folder? you can't do a "relative import", always go from the root using the `./` syntax